modules/users/tests/server/user.server.model.tests.js

```diff
@@ -345,36 +345,37 @@ describe('User Model Unit Tests:', function () {
       
     });
 
-    it('should not allow double dotted characters in email address - "abcdef@abc..com"', function (done) {
```

Comment:
@lirantal Thanks for clarifying that. I wasn't looking at it in that way. You're forcing the assertion to fail, since you have the `if else` block. I hadn't realized what your meant by "Major fixing.." until the other day when I was writing some tests.

To make sure I understand what you're refactoring is accomplishing & that I understand it.. let me ask a question that may be an LDO :) .. So with these changes, you're trying to get all the tests to continue to function properly (pass/fail) even when previous tests fail? This is something I ran into with how the tests are setup currently.